### ISSUE TITLE
Dropdown does not honor orientation, displays in Portrait when phone is in Landscape

### ISSUE THREAD

**Issue by ayercm** [n4kz/react-native-material-dropdown#55]
Everything works fine in portrait orientation.

When I rotate the device, the StackNavigator I am using rotates just fine, so do the material dropdown fields, but when I try to change the selection, the view rotates back to portrait. Is landscape operation supported?

**Comment by isthaison** [comment]
Me too

**Comment by isthaison** [comment]
```
  <Modal    **supportedOrientations={['portrait', 'landscape']}**
 visible={modal} transparent={true} onRequestClose={this.onClose}>
          <TouchableWithoutFeedback onPress={this.onClose}>
            <View style={overlayStyle}>
              <Animated.View
                style={[styles.picker, pickerStyle, pickerStyleOverrides]}
              >
                <ScrollView
                  ref={this.updateScrollRef}
                  style={styles.scroll}
                  scrollEnabled={visibleItemCount < itemCount}
                  contentContainerStyle={styles.scrollContainer}
                >
                  {this.renderItems()}
                </ScrollView>
              </Animated.View>
            </View>
          </TouchableWithoutFeedback>
        </Modal>
```


/components/dropdown/index.js
good luck

**Comment by n4kz** [comment]
Thanks for issue and sorry for delayed reply. Released fix in `0.7.2`. More details are available in RN issue facebook/react-native#11036.
